tests/standalone-airgap/README.md

```diff
@@ -0,0 +1,21 @@
+# TEST: Standalone - Mounted Disk Terraform Enterprise
+
+## About this test
+
+This test for Terraform Enterprise creates a TFE
+installation with the following traits.
+
+- Standalone mode
+- Mounted Disk operational mode
```

Comment:
Most of the content of this test seems to indicate it is for airgap, not mounted disk.